Gemfile

```diff
@@ -45,6 +45,7 @@ gem 'warden-basic_auth', '~> 0.2.1'
 gem 'rails_autolink', '~> 1.1.6'
 gem "will_paginate", '~> 3.0'
 gem "acts_as_list", "~> 0.3.0"
+gem 'acts_as_countable', git: "https://github.com/finnlabs/acts_as_countable.git"
```

Comment:
I'd probably tag the current thing as `v1.0` or `v0.1` and use the tag? Or is that a bad idea?